src/Mobile/BDMobileValidator.php

```diff
@@ -47,4 +48,15 @@ public static function getValidated(string $number): string
 
         return BDMobileFormatter::format($number);
     }
+
+    /**
+     * @param string $number
+     * @return bool
+     */
+    public static function isContainNumberWithoutBDCountryCode(string $number) : bool
```

Comment:
- Function name is not well formed. Should start with does as it is what grammar suggests. Can be something like `doesContainOnlyNumbersAndCountryCode`
- Is there any need to make the function public?